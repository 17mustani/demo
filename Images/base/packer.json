{

"builders": [{
  "type": "amazon-ebs",
  "region": "us-west-2",
  "source_ami_filter": {
    "filters": {
      "virtualization-type": "hvm",
      "name": "*CentOS Linux 7 x86_64 HVM EBS*",
      "root-device-type": "ebs" },
      "most_recent": true
  },
  "instance_type": "c4.2xlarge",
  "ssh_username": "centos",
  "ssh_pty" : false,
  "ami_name": "cncfbase{{timestamp}}",
  "user_data_file": "disable_tty.sh"
}],

"provisioners": [{
      "type": "ansible",
      "playbook_file": "playbook.yml",
      "user": "centos",
      "sftp_command": "/usr/libexec/openssh/sftp-server",
      "extra_arguments": [ "-vvv", "--extra-vars", "packer=yes" ]
  }]

}
