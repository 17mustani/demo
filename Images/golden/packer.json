{

"builders": [{
  "type": "amazon-ebs",
  "region": "us-west-2",
  "source_ami_filter": {
    "filters": {
      "virtualization-type": "hvm",
      "name": "cncfbase*",
      "root-device-type": "ebs"
    },
  "owners": ["750548967590"],
  "most_recent": true
  },
  "instance_type": "c4.2xlarge",
  "ssh_username": "centos",
  "ssh_pty" : false,
  "ami_name": "cncf{{timestamp}}",
  "user_data_file": "disable_tty.sh"
}],

"provisioners": [{
      "type": "ansible",
      "playbook_file": "playbook.yml",
      "user": "centos",
      "sftp_command": "/usr/libexec/openssh/sftp-server",
      "extra_arguments": [ "--extra-vars", "packer=yes" ]
  }]

}
